deck/templatetags/deck_tags.py

```diff
@@ -19,7 +19,10 @@ def allowed_to_vote(user, proposal):
 
 @register.filter
 def get_rate_display(user, proposal):
-    return proposal.votes.get(user=user).get_rate_display()
+    if not isinstance(user, AnonymousUser):
```

Comment:
Actually here I'm considering it is better to use:

```
if user.is_registered(): ...
```

According to documentation is the same as:

```
if not user.is_anonymous(): ...
```

https://docs.djangoproject.com/en/1.7/ref/contrib/auth/#django.contrib.auth.models.User.is_authenticated